code/game/atoms_movable.dm

```diff
@@ -267,6 +267,10 @@
 		src.thrower = null
 		src.throw_source = null
 
+	if(ishuman(thrower)) //Human specific procs
+		var/mob/living/carbon/human/H = thrower
+		H.camo_off_process(SCOUT_CLOAK_OFF_ATTACK)
```

Comment:
having this called every time a human throws something feels like a bad idea.

Reply:
Agreed, unless if the proc is really cheap. Most cloaking procs are pretty intrusive this way.
Suggestions on alternatives?